utils/symbolicate-linux-fatal.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+
+# TODOs:
+# * verbose output
+# * search symbols by name for the not <unavailable> ones
+
+import sys
+import lldb
```

Comment:
Oops, was signed in with wrong account (just above).

Last note: I'm more than happy to rewrite with `addr2line` but seems like less promising approach.